client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -66,7 +67,12 @@ class SpeedUpSiteSettings extends Component {
 									'files and images so your visitors enjoy ' +
 									'the fastest experience regardless of device or location.'
 							) }
-							link="http://jetpack.com/support/site-accelerator/"
+							link={
+								siteIsAtomic
+									? 'https://wordpress.com/support/settings/performance-settings/#enable-site-accelerator'
+									: 'https://jetpack.com/support/site-accelerator/'
+							}
+							privacyLink={ ! siteIsAtomic }
```

Comment:
Privacy links [shouldn't be touched](https://github.com/Automattic/wp-calypso/issues/59976#:~:text=the%20links%20for%20Privacy%20Information%20can%20stay%20as%20is). To keep things as they were, I think this should be
```suggestion
							privacyLink="https://jetpack.com/support/site-accelerator/#privacy"
```

Reply:
Thanks, I misread that in the GH issue. The link is already set, I just removed the prop that was setting it as false. 👍